project2/src/main/java/com/revature/moneyRequest/MoneyRequestMicroservice.java

```diff
@@ -0,0 +1,9 @@
+package com.revature.moneyrequest;
```

Comment:
Shouldn't this be `.moneyRequest` to match the rest of the refactoring?

Reply:
yup